src/app/fdctl/run/tiles/fd_poh.c

```diff
@@ -700,9 +700,11 @@ publish_became_leader( fd_poh_ctx_t * ctx,
   uchar * dst = (uchar *)fd_chunk_to_laddr( ctx->pack_out_mem, ctx->pack_out_chunk );
 
   fd_became_leader_t * leader = (fd_became_leader_t *)dst;
-  leader->slot_start_ns = slot_start_ns;
-  leader->bank = ctx->current_leader_bank;
+  leader->slot_start_ns           = slot_start_ns;
```

Comment:
Do we still need to pass slot_start_ns?

Reply:
No, but it might be helpful for metrics in the future, so I left it